conans/client/generators/text.py

```diff
@@ -11,6 +11,13 @@
 from conans.util.log import logger
 
 
+class OrderedDefaultDict(OrderedDict):
+
+    def __missing__(self, key):
+        self[key] = value = defaultdict(dict)
```

Comment:
why not just `return self[key] = defaultdict(dict)`? I don't get the point of `value` variable